aea/manager.py

```diff
@@ -37,6 +38,23 @@
 from aea.crypto.helpers import create_private_key
 
 
+class MAMState:
```

Comment:
I think a function or helper method to convert to dict is enough, we don't need this object instantiated as far as I can tell.

Reply:
Fine. Done.